tests/test_device_validator.py

```diff
@@ -266,3 +268,49 @@ def test_finite_diesel_generator(self):
         self.assertIsNone(validate_finite_diesel_generator(energy_rate=100))
         with self.assertRaises(D3ADeviceException):
             validate_finite_diesel_generator(energy_rate=-1)
+
+
+class TestHomeMeterValidator(unittest.TestCase):
+    """Tests for the HomeMeterValidator class."""
+
+    @classmethod
+    def setUpClass(cls) -> None:
```

Comment:
No need for the return type hint imo

Reply:
Why not? Actually, that's also automatically filled up by PyCharm.

Reply:
Type hints add more verbosity to the code, but they help to spot mistakes while coding in IDEs or while using static type checkers. I would actually encourage doing it everywhere in the code, when possible.